src/Miunie.Discord/Services/HelpService.cs

```diff
@@ -0,0 +1,129 @@
+﻿// This file is part of Miunie.
+//
+//  Miunie is free software: you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License as published by
+//  the Free Software Foundation, either version 3 of the License, or
+//  (at your option) any later version.
+//
+//  Miunie is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+//  GNU General Public License for more details.
+//
+//  You should have received a copy of the GNU General Public License
+//  along with Miunie. If not, see <https://www.gnu.org/licenses/>.
+
+using Discord.Commands;
+using Discord.WebSocket;
+using Miunie.Core.Entities;
+using Miunie.Core.Providers;
+using Miunie.Discord.Attributes;
+using Miunie.Discord.Embeds;
+using Miunie.Discord.Entities;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Miunie.Discord
+{
+    public class HelpService
```

Comment:
This is not really a `Service`. We would confuse the `Core` Services with these.

I think a better fit would be `CommandHelpProvider`. Also because the word `Help` might be a bit too general. 👍

Reply:
Also if you called it a `Service` because of  `CommandService`, keep in mind that comes from `Discord.Net` and has nothing to do with hour architecture. I can see how it may look confusing. 😅

Reply:
No, it makes sense. I actually left it as `HelpService`, because I couldn't think of a proper name for it at the time.